### ISSUE TITLE
Favicon support

### ISSUE THREAD

**Issue by mojombo** [redwoodjs/redwood#49]
It would be nice to have proper Favicon support, let's see if we can find a library that works properly.

**Comment by cannikin** [comment]
In the meantime I tried including `<link rel="icon" type="image/png" href="/favicon.png" />` in `index.html` but we don't have any way to serve static assets. :(

**Comment by mojombo** [comment]
Try including it in `index.js`:

```js
import from './favicon.png'
```

That should get Webpack to include it.

**Comment by cannikin** [comment]
Yeah I tried that, no go. :( For stylesheets it actually transforms them into being inline `<style>` tags but in this case we're adding a <link> which makes a separate server request. Webpack doesn't know anything about that in dev mode unless you give it a directory to serve static assets from, I think?

**Comment by peterp** [comment]
I think you can add this:
```js
new HtmlWebpackPlugin({
    favicon: "./src/favicon.png"
})
```

Over here: https://github.com/redwoodjs/redwood/blob/master/packages/scripts/config/webpack.config.js#L70-L72

We could also create a `static/` or `public/` folder that gets copied during build time, using [copy-webpack-plugin](https://webpack.js.org/plugins/copy-webpack-plugin/), which might give us the most mileage.

**Comment by peterp** [comment]
Closed via #107
